src/Components/ModelGroupLabel/ModelGroupLabel.tsx

```diff
@@ -30,26 +30,29 @@ export const ModelGroupLabel: React.FC<ModelGroupLabelProps> = ({
 
     const [showContextMenu, setShowContextMenu] = useState(false);
     const [contextItems, setContextItems] = useState<IContextualMenuItem[]>([]);
-    const id = 'ID' + createGUID();
+    const [groupId, setGroupId] = useState('id');
```

Comment:
I'm having a little trouble testing this since the Globe SceneView story still doesn't group together close markers (at least on this branch) ?